cypress/support/commands.ts

```diff
@@ -156,6 +157,101 @@ Cypress.Commands.add(
   },
 );
 
+Cypress.Commands.add(
+  'addProblemsToContest',
+  ({
+    contestAlias,
+    problems,
+  }) => {
+    cy.visit(`contest/${contestAlias}/edit/`);
+    cy.get('a[data-nav-contest-edit]').click();
+    cy.get('a.dropdown-item.problems').click();
+
+    for (const idx in problems) {
+      cy.get('input[type="text"]').type(problems[idx].problemAlias)
+      cy.get('.typeahead-dropdown').click();
+      cy.get('.add-problem').click();
+    }
+  },
+);
+
+Cypress.Commands.add(
+  'changeAdmissionModeContest',
+  ({
+    contestAlias,
+    admissionMode,
+  }) => {
+    cy.visit(`contest/${contestAlias}/edit/`);
+    cy.get('a[data-nav-contest-edit]').click();
+    cy.get('a.dropdown-item.admission-mode').click();
+    cy.get('select[name="admission-mode"]').select(
+      admissionMode,
+    ); // private | registration | public
+    cy.get('.change-admission-mode').click();
+  },
+);
+
+Cypress.Commands.add(
+  'enterToContest',
```

Comment:
nit: en inglés rara vez se usa "enter to": el "enter" ya implica el "a el".

```suggestion
  'enterContest',
```

Reply:
Listo.